packages/rum-core/src/domain/rumEventsCollection/action/clickChain.ts

```diff
@@ -85,23 +85,40 @@ function mouseEventDistance(origin: MouseEvent, other: MouseEvent) {
 
 function finalizeClicks(clicks: Click[], rageClick: Click) {
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏I love it! Having the frustration logic for rage and other types in a single place is awesome.